frame/staking/src/lib.rs

```diff
@@ -2159,15 +2150,15 @@ decl_module! {
 		/// transaction in the block.
 		///
 		/// # <weight>
-		/// See `crate::weight` module.
+		/// The weight of this call is not enforced by the system module and is enforced in this
+		/// module directly.
```

Comment:
maybe you can add that this call weight must not exceed OffchainSolutionWeightLimit in order to be valid

Reply:
outdated.